tests/pos/opaque-inline.scala

```diff
@@ -0,0 +1,20 @@
+
+object refined:
+  opaque type Positive = Int
+
+  object Positive extends PositiveFactory
+
+  trait PositiveFactory:
+    inline def apply(value: Int): Positive = value
+
+    def f(x: Positive): Positive = x
+    inline def fapply(value: Int): Positive =
+      val vv = (value, value) // error: implementation restriction
```

Comment:
is this comment correct? Is this still an error? I'm confused about it being in `tests/pos`, maybe it is checking an error message...

Reply:
usually this marks something that was previously an error, which was fixed in the PR that added the file